CONTRIBUTING.md

```diff
@@ -0,0 +1,228 @@
+# Contributing Code & Datasets
+
+## Installing and running tests
+
+First, check out the repository from Github.
+
+Then, install [pyenv](https://github.com/pyenv/pyenv#installation) to manage your Python versions.
+You'll want to install Python 2.7.11, and the latest versions of Python 3.6 and 3.7
+
+Once pyenv and the three versions of Python are installed, install [tox](https://tox.readthedocs.io/en/latest/), our test runner.
+
+Finally, run tox with `tox`.  All tests should pass!
+
+
+## Contributing a new dataset loader
+
+To add a new dataset loader:
+1. Create an index in `mirdata/indexes`, e.g. `my_dataset_index.json`
```

Comment:
1. Create a script in `scripts`, e.g. `make_my_dataset_index.py`, which generates an index file (see below)
2. Run the script on the canonical* version of the dataset and save the index in `mirdata/indexes` e.g. `my_dataset_index.json`.
3. Create a module in `mirdata`, e.g. `my_dataset.py`

*Whenever possible, this should be the official release of the dataset as published by the dataset creators. When this is not possible, (e.g. for data that is no longer available), find as many versions of the data as you can from different researchers (at least 4), and use the most common one.

Reply:
^^ @keunwoochoi @magdalenafuentes controversial stuff in the *comment. What do you think?

Reply:
🚀